pkgs/development/libraries/clFFT/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, fftw, fftwFloat, boost166, opencl-clhpp, ocl-icd }:
+
+let
+  version = "2.12.2";
+in stdenv.mkDerivation {
+  pname = "clFFT";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "clMathLibraries";
+    repo = "clFFT";
+    rev = "ce107c4d5432d70321af8980a5e7fb64c4b4cce4";
+    sha256 = "134vb6214hn00qy84m4djg4hqs6hw19gkp8d0wlq8gb9m3mfx7na";
+  };
+
+  postPatch = ''
+    cd src
+  '';
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [ fftw fftwFloat boost166 opencl-clhpp ocl-icd ];
+
+  meta = with stdenv.lib; {
+    description = "Library containing FFT functions written in OpenCL. In addition to GPU devices, the library also supports running on CPU devices to facilitate debugging and heterogeneous programming";
+    license = licenses.asl20;
+    homepage = http://clmathlibraries.github.io/clFFT/ ;
+    platforms = platforms.linux;
+    maintainers = with stdenv.lib.maintainers; [ chessai ];
```

```suggestion
    maintainers = with maintainers; [ chessai ];
```
You already set `with stdenv.lib;` above.